### PR TITLE
chore: ME design audit CE

### LINKED ISSUE #1969: [Task] Enable currentRow in column action selectors

### PR DESCRIPTION
**Pull request by ChandanBalajiBP**
## Description
https://www.notion.so/appsmith/Multiple-Environments-Design-Audit-f230b300039549fcb2bb784b05ff0c1c

#### PR fixes following issue(s)
Fixes #1969 

- Bug fix (non-breaking change which fixes an issue)

- [x] Manual
- [ ] Jest
- [ ] Cypress
>
>
#### Test Plan

Tested these issues:
[Spacing between headers](https://www.notion.so/appsmith/Multiple-Environments-Design-Audit-f230b300039549fcb2bb784b05ff0c1c?p=4c4e381f307848f5a41036bd21251ff7&pm=s)
[Self signed certificate should be part of Oauth 2](https://www.notion.so/appsmith/Multiple-Environments-Design-Audit-f230b300039549fcb2bb784b05ff0c1c?p=5b79d2eb17164e7ab13ee13e9128ac55&pm=s)
[Wrong CTA for API datasources](https://www.notion.so/appsmith/Multiple-Environments-Design-Audit-f230b300039549fcb2bb784b05ff0c1c?p=2962f718df7f4cc4a11515b4bee64b89&pm=s)
[The info banner on git modal](https://www.notion.so/appsmith/Multiple-Environments-Design-Audit-f230b300039549fcb2bb784b05ff0c1c?p=659b03d6f7644150921385cc001d9c09&pm=s)
[Code split issues](https://theappsmith.slack.com/archives/CPG2ZTXEY/p1692867977713909)

>
>
#### Issues raised during DP testing
https://github.com/appsmithorg/appsmith-ee/pull/1970#issuecomment-1698646837
>
>
>
#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [x] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [x] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/ee/components/EnvDeployInfoModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function EnvDeployInfoModal() {
+  return null;
```
Comment (ankitakinger) [inline app/client/src/ee/components/EnvDeployInfoModal/index.tsx:2]: this code should exist at the below path:
`app/client/src/ce/components/EnvDeployInfoModal/index.tsx`

The code for this file should be just:
`export * from "ce/components/EnvDeployInfoModal"`

**Review by ankitakinger — APPROVED** [review]
LGTM

File: app/client/src/ce/constants/messages.ts [hunk 2]
```diff
@@ -1768,6 +1768,8 @@ export const NEW_API_BUTTON_TEXT = () => "New API";
 export const GENERATE_NEW_PAGE_BUTTON_TEXT = () => "Generate new page";
 export const RECONNECT_BUTTON_TEXT = () => "Reconnect";
 export const SAVE_BUTTON_TEXT = () => "Save";
+export const SAVE_AND_AUTHORISE_BUTTON_TEXT = () => "Save and Authorize";
```
Comment (ayushpahwa) [inline app/client/src/ce/constants/messages.ts:1771]: I think these messages are already present, I can see the same on line 1774

File: app/client/src/pages/common/datasourceAuth/index.tsx [hunk 3]
```diff
@@ -20,7 +20,9 @@ import {
   CANCEL,
   OAUTH_AUTHORIZATION_APPSMITH_ERROR,
   OAUTH_AUTHORIZATION_FAILED,
+  SAVE_AND_AUTHORISE_BUTTON_TEXT,
```
Comment (ayushpahwa) [inline app/client/src/pages/common/datasourceAuth/index.tsx:23]: This import is done twice here

**Review by ayushpahwa — CHANGES_REQUESTED** [review]

**Comment by ChandanBalajiBP** [comment]
./ok-to-test

**Review by ayushpahwa — APPROVED** [review]

**Comment by ChandanBalajiBP** [comment]
This PR was tested in [EE PR](https://github.com/appsmithorg/appsmith-ee/pull/1970) and here is the [comment](https://github.com/appsmithorg/appsmith-ee/pull/1970#issuecomment-1702734755) for approval. Hence adding test plan label myself.

**Pull request merged**